src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -6479,64 +6483,69 @@ namespace WaterThermalTanks {
 					TempUp = Tank.Node( NodeNum - 1 ).Temp;
 					if ( TempUp < NodeTemp ) InvMixUp = Tank.InversionMixingRate;
 				}
+//BUG? When NodeNum==1 TempUp is 0.0   Shouldn't it be NodeTemp to void the vertical conduction? !!!!!!!!!!!!!!!!!
```

Comment:
@DeadParrot yes [this does look like a bug](https://github.com/NREL/EnergyPlus/commit/9efa39dbb1a6b792575f7bcd0d876cc71a0c5dee#commitcomment-16065959) to me too. I'll dig in more and probably make a unit test here too to verify before changing it. Thanks for finding it.